### PR TITLE
Remove extra `Quantity.fromString()` call

### PR DESCRIPTION
**Pull request by davidxia**
in Quantity constructor. Avoid doing extra work.
We only need to call it once.

### CODE REVIEW

File: kubernetes/src/main/java/io/kubernetes/client/custom/Quantity.java [hunk 1]
```diff
@@ -37,8 +37,9 @@ public Quantity(final BigDecimal number, final Format format) {
     }
 
     public Quantity(final String value) {
```
Comment (davidxia) [inline kubernetes/src/main/java/io/kubernetes/client/custom/Quantity.java]: Aside from this PR, it seems weird to me that this constructor calls a static factory method, gets an instance of Quantity, and reassigns the attributes to itself.

**Comment by brendandburns** [comment]
Needs rebase now.

**Comment by davidxia** [comment]
Thanks. Rebased.

**Pull request merged**